src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] reportFromFile = getReportFromFile(fromFileName);
+        WorkWithFile report = new WorkWithFile();
+        report.writeReportToFile(reportFromFile, toFileName);
+    }
+
+    private String[] getReportFromFile(String fileName) {
```

Comment:
[Divide program logic](https://github.com/mate-academy/jv-work-with-file/blob/master/checklist.md#divide-program-logic)
it' better to create readFromFile and createReport separate methods